github/commitstatus.go

```diff
@@ -31,6 +32,12 @@ func (e *StatusError) Error() string {
 // API is the GitHub API endpoint.
 const API = "https://api.github.com"
 
+// Maximum number of retries for the retryable http request
+const maxRetries = 3
```

Comment:
You are using lower camelcase here while the variable at line 39 is upper. Is there any reason about that?

Reply:
Yes, tests are in different package so I wanted `MaxSleepTime` to be overridable from tests...

It turns out it was much simpler to achieve what I wanted:
```
// MaxSleepTime = 15 minutes (900s)
rateLimitReset: []int{now + 901},
```

Since the test I wanted to include did not really bring any value I removed it and did not include it in this PR. So to sumarize, yes I can rename already `MaxSleepTime` to `maxSleepTime` and if at some later time I need to override it from the tests, I will change what is needed then...

Reply:
renamed the other value....